internal/schedule_client.go

```diff
@@ -363,6 +363,10 @@ type (
 		//
 		// [Visibility]: https://docs.temporal.io/visibility
 		TypedSearchAttributes SearchAttributes
+
+		// KeepOriginalWorkflowID - If true, and the action would start a workflow, a timestamp will not be
+		// appended to the scheduled workflow id.
+		KeepOriginalWorkflowID bool
```

Comment:
What would happen if users try to set this to true and call `CreateSchedule` on an older server version that doesn't support this field? I suspect it will succeed, but ignore the call fields correct?

Reply:
I think you are right. Protobuf allows us to ignore fields which we do not recognize. So server will just ignore the new field.

Reply:
We should probably document this option may not apply depending on the server version, ideally say what server version this option will be respected